app/jobs/iiif_print/child_works_from_pdf_job_decorator.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
```

Comment:
This looks to be something that may make sense to contribute back to IIIF Print.